src/sid/initial_conditions.py

```diff
@@ -194,7 +194,7 @@ def sample_initial_distribution_of_infections_and_immunity(
         virus_strains (Dict[str, Any]): A dictionary with the keys ``"names"`` and
             ``"factors"`` holding the different contagiousness factors of multiple
             viruses.
-        vaccination_model (Optional[Callable]): A function accepting ``states``,
+        vaccination_models (Optional[Callable]): A function accepting ``states``,
             ``params``, and a ``seed`` which returns boolean indicators for individuals
             who received a vaccination.
```

Comment:
I think you forgot to update the docstring here.

Reply:
Done